File: Java/Basicos/src/ListaConNexo/Nodo.java
package ListaConNexo;
public class Nodo {
	private double dato;
	private Nodo anterior;
	private Nodo siguiente;
	
	public Nodo(double dato) {
		this.dato = dato;
		anterior = null;
		siguiente = null;
	}

	public void setDato(double dato) {
		this.dato = dato;
	}

	public void setAnterior(Nodo anterior) {
		this.anterior = anterior;
	}

	public void setSiguiente(Nodo siguiente) {
		this.siguiente = siguiente;
	}

	public double getDato() {
		return dato;
	}

	public Nodo getAnterior() {
		return anterior;
	}

	public Nodo getSiguiente() {
		return siguiente;
	}
}

File: C++/Clase 4/1/Nodo.cpp
#include "Nodo.h"
Nodo::Nodo(double _dato){
    dato = _dato;
    siguiente = NULL;
};
double Nodo::getDato(){
    return dato;
};
Nodo* Nodo::getSiguiente(){
    return siguiente;
};
void Nodo::setSiguiente(Nodo* _siguiente){
    siguiente = _siguiente;
};
